scripts/image-builder/config/kickstart.ks.template

```diff
@@ -81,4 +81,33 @@ for uuid in $(nmcli -f uuid,type,autoconnect connection | awk '$2 == "ethernet"
     sed -i '/autoconnect=.*/d' "${file}"
 done
 
+# Configure an empty local ostree repository and server if the "edge" remote points to "localhost"
+if grep -q '^url=http://127.0.0.1:8080' /etc/ostree/remotes.d/edge.conf ; then
+    OSTREE_REPO=/var/lib/ostree-server-local
+    OSTREE_SERV=ostree-server-local.service
+
+    # Create an empty repository with a summary
+    mkdir -p ${OSTREE_REPO}/repo
+    ostree --repo=${OSTREE_REPO}/repo init --mode=archive-z2
+    ostree summary --repo ${OSTREE_REPO}/repo --update
+
+    # Create an HTTP server for the repository
+    cat > /etc/systemd/system/${OSTREE_SERV} <<EOF
+[Unit]
+Description=Python HTTP server for the local ostree repository
+
+[Service]
+User=root
+WorkingDirectory=${OSTREE_REPO}
+ExecStart=/usr/bin/python -m http.server 8080
```

Comment:
https://www.osbuild.org/guides/user-guide/building-ostree-images.html#setting-up-an-http-server

> Note: You might be wondering why to bother with a container when you can just use "python -m http.server". The problem is that OSTree produces way too many requests and the Python HTTP server simply fails to keep up with OSTree

Reply:
Is this documented somewhere that Python server is overwhelmed by the ostree requests?
I would like to avoid complicating the current solution if possible.

Reply:
I switched to using Caddy as an HTTP server. Using Apache HTTPD is too complex for the configuration and I wanted to avoid running containers for this.